examples/react-dashboard/package.json

```diff
@@ -12,6 +12,7 @@
     "recharts": "^2.1.13"
   },
   "scripts": {
+    "dev": "react-scripts start",
```

Comment:
Basically just aliasing `"dev"` to whatever `"start"` is for these examples. This makes them work nicely with Turborepo when linked locally, so Turbo triggers the necessary (local) builds first.